daemon/cmd/daemon_main.go

```diff
@@ -1180,6 +1181,7 @@ func initEnv(cmd *cobra.Command) {
 	// useful if the daemon is being round inside a namespace and the
 	// BPF filesystem is mapped into the slave namespace.
 	bpf.CheckOrMountFS(option.Config.BPFRoot, k8s.IsEnabled())
+	cgroups.CheckOrMountCgrpFS(option.Config.CGroupRoot)
```

Comment:
any reason why can't this be on line 584 of daemon/cmd/daemon.go?

Reply:
This is part of a revert commit.